common4j/build.gradle

```diff
@@ -219,7 +227,9 @@ dependencies {
     testAnnotationProcessor "org.projectlombok:lombok:$rootProject.ext.lombokVersion"
     testImplementation "junit:junit:$rootProject.ext.junitVersion"
     testImplementation "org.mockito:mockito-core:$rootProject.ext.mockitoCoreVersion"
-
+    testImplementation "io.mockk:mockk:1.11.0"
+    testImplementation "org.mockito:mockito-inline:$rootProject.ext.mockitoCoreVersion"
+    testImplementation "org.mockito.kotlin:mockito-kotlin:4.1.0"
```

Comment:
Please define these versions in the version file

Reply:
Done

Reply:
I will have to use the version numbers instead of variables otherwise the build for adal and broker will fail. I don't have privileges to modify the gradle version files for those two repositories. I can add the variables in android-complete but not the other two repositories.